### PR TITLE
Point to GitHub Releases in CHANGELOG.md

### PR DESCRIPTION
**Pull request by strugee**
A lot of people will check here and not find anything (see e.g. #1633). We should point them to the right place.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The change log is kept in [GitHub Releases](https://github.com/Homebrew/brew/releases).
```
Comment (MikeMcQuaid) [inline CHANGELOG.md]: `See Homebrew's [releases on GitHub](https://github.com/Homebrew/brew/releases) for the changelog.`

**Comment by MikeMcQuaid** [comment]
Thanks for this! A suggested language tweak but I agree this seems like a good idea and thanks for being the change you want to see in the world 👍

**Comment by strugee** [comment]
@MikeMcQuaid fixed, thanks! Your version is definitely better

> thanks for being the change you want to see in the world 

This is probably the best review comment I've ever received. Thanks for making this community awesome <3

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew, @strugee! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**